api/src/utils/env.ts

```diff
@@ -34,6 +34,7 @@ assert.ok(process.env.FCC_ENABLE_DEV_LOGIN_MODE);
 if (process.env.FREECODECAMP_NODE_ENV !== 'development') {
   assert.ok(process.env.PORT);
   assert.ok(process.env.MONGOHQ_URL);
+  assert.ok(process.env.SENTRY_DSN);
```

Comment:
```suggestion
  assert.ok(process.env.SENTRY_DSN);
  assert.notEqual(
    process.env.SENTRY_DSN,
    'dsn_from_sentry_dashboard',
    'The DSN from Sentry's dashboard should be used.'
  )
```
It may seem a little excessive, but the goal for this module is to throw an error whenever an environment variable is not right. Either if it's missing, not an allowed value or is a specifically disallowed value (like in this case).

The hope is that anyone seeing errors from this module will immediately see what's wrong and how to fix it.